cli/enginecli/core.py

```diff
@@ -516,13 +498,15 @@ def lootbox_gogogo(
     )
 
     print("Setting pool controller...")
-    terminus_contract.set_pool_controller(
-        admin_token_pool_id, lootbox_contract.address, tx_config
+    admin_terminus_contract.set_pool_controller(
+        admin_terminus_pool_id, lootbox_contract.address, tx_config
     )
 
     contracts = {
         "Lootbox": lootbox_contract.address,
-        "adminTokenPoolId": admin_token_pool_id,
+        "TerminusAddress": terminus_address,
```

Comment:
`TerminusAddress`, `AdminTerminusAddress`, and `AdminTokenPoolId` almost certainly do not belong at the root level of a dictionary called `contracts. I see that you are going off of the previous code, though, so no need to change this.